pkg/tbtc/dkg.go

```diff
@@ -28,7 +28,7 @@ const (
 	// is used to calculate the submission delay period that should be respected
 	// by the given member to avoid all members submitting the same DKG result
 	// at the same time.
-	dkgResultSubmissionDelayStepBlocks = 15
+	dkgResultSubmissionDelayStepBlocks = 3
```

Comment:
I am concerned this is too low. The gas price bump happens after [one minute by default](https://github.com/keep-network/keep-common/blob/f89349433020ed33f9cff2eafc07e97dbeef32aa/pkg/chain/ethereum/ethutil/mining_waiter.go#L19) so if the initial estimation was not successful, there will be not enough time to do even one price bump.

Reply:
I tested that yesterday on Sepolia where gas prices were quite high and volatile. Haven't observed any problem so I think we should be good. It is important to reduce DKG timeout to the minimum and this factor strongly commits to that value. We will monitor the situation on mainnet and increase that if necessary.

Reply:
Worth noting that the price bump will be actually done. The only downside is that the next member may front-run with their own transaction. This may lead to a collision sometimes but is not harmful to the protocol.